src/SupportMessage/Collection.tsx

```diff
@@ -3,86 +3,75 @@ import styled from 'styled-components'
 
 import { Text } from '../Text'
 import { Box } from '../Box'
-import { SupportMessage } from './SupportMessage'
+import { SupportMessage, SupportMessageProps } from './SupportMessage'
+
+const variationList: SupportMessageProps[] = [
+  {
+    type: 'info',
+    title: 'Additional information',
+    description: 'Information needs to be communicated to the user.',
+  },
+  {
+    type: 'info-outline',
+    title: 'Additional information',
+    description: 'Information needs to be communicated to the user.',
+  },
+  {
+    type: 'alert',
+    title: 'Potential issue',
+    description: 'The user should be aware and possibly exercise caution.',
+  },
+  {
+    type: 'warning',
+    title: 'Error encountered',
+    description: "An error has occurred, here's feedback on how to proceed.",
+  },
+]
 
 export const CollectionPage: FC = () => {
   return (
     <Box flex direction="column">
-      <ContainerBox>
-        <Label tag="span" typo="header-small">
-          Type info-outline
-        </Label>
-        <SupportMessage
-          type="info-outline"
-          description="Info support message"
-        />
-      </ContainerBox>
-
-      <ContainerBox>
-        <Label tag="span" typo="header-small">
-          Type info - Default
-        </Label>
-        <SupportMessage type="info" description="Info support message" />
-      </ContainerBox>
-
-      <ContainerBox>
-        <Label tag="span" typo="header-small">
-          Type info - With title
-        </Label>
-        <SupportMessage
-          type="info"
-          title="General info title"
-          description="Info support message"
-        />
-      </ContainerBox>
-
-      <ContainerBox>
-        <Label tag="span" typo="header-small">
-          Type alert - Default
-        </Label>
-        <SupportMessage type="alert" description="Alert support message" />
-      </ContainerBox>
-
-      <ContainerBox>
-        <Label tag="span" typo="header-small">
-          Type alert - With title
-        </Label>
-        <SupportMessage
-          type="alert"
-          title="General alert title"
-          description="Alert support message"
-        />
-      </ContainerBox>
-
-      <ContainerBox>
-        <Label tag="span" typo="header-small">
-          Type warning - Default
-        </Label>
-        <SupportMessage type="warning" description="Warning support message" />
-      </ContainerBox>
-
-      <ContainerBox>
-        <Label tag="span" typo="header-small">
-          Type warning - With title
-        </Label>
-        <SupportMessage
-          type="warning"
-          title="General warning title"
-          description="Warning support message"
-        />
-      </ContainerBox>
+      {variationList.map(({ type, title, description }) => {
```

Comment:
<img width="1001" alt="Screenshot 2022-07-15 at 12 19 52" src="https://user-images.githubusercontent.com/14129033/179214024-0908eb8e-0ff9-4e23-91e4-f2ad15ae7055.png">

I've updated the view a bit and added some more concrete example